cmd/runtimecfg/node-ip.go

```diff
@@ -113,10 +114,11 @@ func set(cmd *cobra.Command, args []string) error {
 		nodeIPs += "," + chosenAddresses[1].String()
 	}
 	remoteWorker := false
-	// if chosen ip doesn't match vips need create a file that
-	// this file will be used by keepalived container to verify if it should run or not
-	// We want to disable keepalive in case this host is remote worker
-	if len(vips) > 0 && !matchesVips {
+	// if chosen ip doesn't match vips, we need create a file that
+	// will be used by keepalived container to verify if it should run or not
+	// We want to disable keepalived in case this host is remote worker
+	// This is skipped in case of user managed load balancer as the VIPs are not managed by keepalived.
+	if len(vips) > 0 && !matchesVips && !userManagedLB {
```

Comment:
This is not the right place, this one ensures that keepalived will sleep forever and not remote-worker label